Appraisals

```diff
@@ -33,5 +33,8 @@ end
 
 appraise 'rails-5.2' do
   gem 'rails', '~> 5.2.0'
-  gem 'pg'
```

Comment:
Is this by default in rails >=5.2 ?

Reply:
This was needed before to override 

```
  spec.add_development_dependency "pg", "~> 0.21"
```

and allow Rails 5.2 Appraise to use `pg` 1.x

Now it is not needed anymore because `pg` dependency on the gemspec gem has been relaxed